vcpkg.json

```diff
@@ -1,39 +1,21 @@
 {
     "name": "adobe-source-libraries",
     "version": "1.0.45",
```

Comment:
Does that version number need to be updated when we do a release? Is that document?

Reply:
I have no idea. I took that line verbatim from [what's in `main`](https://github.com/stlab/adobe_source_libraries/blob/main/vcpkg.json#L3) today. I'd ask @camio for details there.